File: VRP-Core/src/main/java/VRPCore/Models/Fee.java
package VRPCore.Models;

import VRPCore.Enums.FeeInterval;
import VRPCore.Enums.FeeTypes;

public class Fee {
    public String FeeName;
    public long LastPaid;
    public FeeTypes FeeType;
    public FeeInterval FeeInterval;
}

File: VRP-Core/VRP-Core/src/main/kotlin/Models/Fee.kt
package VRPCore.Models

import VRPCore.Enums.FeeInterval
import VRPCore.Enums.FeeType

class Fee {
    var FeeName: String? = null
    var LastPaid: Long = 0
    var FeeType: FeeType? = null
    var FeeInterval: FeeInterval? = null
}